### PR TITLE
chore: create hebrew tralnsation structure for cloudevents spec

### LINKED ISSUE #1042: translation: cloudevents spec to hebrew
I want to translate the CloudEvents spec to Hebrew. 

The issues I think I will encounter is RLT support from github

Never done this before but I think this will be cool experience

### PR DESCRIPTION
**Pull request by sasha-tkachev**
Signed-off-by: Alexander Tkachev <sasha64sasha@gmail.com>

Fixes #1042 

As required by the translations spec, I created an empty Hebrew translation structure.

### CODE REVIEW

**Comment by duglin** [comment]
@sasha-tkachev let me know when you think this one is ready for review

**Comment by duglin** [comment]
Looks like there's a bad link:
```
./cloudevents/languages/he/translations.md: Can't find: ./cloudevents/languages/he/../../translations.md
```

**Comment by sasha-tkachev** [comment]
@duglin  it is ready

**Comment by duglin** [comment]
Can you fix the bad link as noted in: https://github.com/cloudevents/spec/pull/1046#issuecomment-1216732734

File: cloudevents/languages/he/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+מסמך זה תרם תורגם. בבקשה תשתמשו [בגרסה האנגלית של המסמך](../../README.md) לבינתיים.
```
Comment (duglin) [inline cloudevents/languages/he/README.md:1]: Google translates this as:
This contributed document has been translated. Please use [the English version of the document](https://github-com.translate.goog/cloudevents/spec/blob/4f1e496fef4383a70fb2d859fd47a8b7ec99afb4/cloudevents/README.md?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp) for now.

Shouldn't it be "**not** translated" ? I suspect the others have the same missing "not"

Comment (sasha-tkachev) [reply cloudevents/languages/he/README.md:1]: Oops, one char typo. Nice catch

**Comment by duglin** [comment]
LGTM

**Comment by duglin** [comment]
Approved on the 8/18 call

**Pull request merged**